### PR TITLE
New Rule: Link: Google Translate

### PR DESCRIPTION
**Pull request by ajpc500**

### CODE REVIEW

File: detection-rules/link_google_translate.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: "Link: Google Translate"
+description: |
+  Attackers have used the Google Translate service to deliver links to malicious sites repackaged with a translate.goog top-level domain.
+  This rule identifies instances of Google Translate links from unsolicited senders.
+references:
+  - "https://www.kaspersky.com/blog/google-translate-scheme/46377/"
+type: "rule"
+authors:
+  - twitter: "ajpc500"
+severity: "medium"
```
[the remark this answers is not in the document]
Comment (ajpc500) [reply detection-rules/link_google_translate.yml:11]: yep, fair!

File: detection-rules/link_google_translate.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+references:
+  - "https://www.kaspersky.com/blog/google-translate-scheme/46377/"
+type: "rule"
+authors:
+  - twitter: "ajpc500"
+severity: "medium"
+source: |
+  type.inbound 
+  and any(body.links, 
+    strings.iends_with(.href_url.domain.root_domain, "translate.goog")
```
[the remark this answers is not in the document]
Comment (ajpc500) [reply detection-rules/link_google_translate.yml:15]: yeah, not sure why i did that in the first place tbh! hunting around, the suggested fix looks good 👍🏻

**Comment by morriscode** [comment]
Retro results look great on this rule. 1:1 overlap with Free subdomain link with login or captcha (first-time sender) in testing, but a good complimentary rule.

**Review by morriscode — APPROVED** [review]
👍

**Pull request merged**